### PR TITLE
Made changes to fix EMPT107

### PR DESCRIPTION
**Pull request by Parth59**
Hi Team,

This change is for fixing EMPT107. I have performed minor test runs to the best of my abilities.

Link to ticket
RA-1875 (https://issues.openmrs.org/browse/RA-1875)
@isears 

Steps to reproduce :

i) Login in as admin inpatient.
ii) Click on the “Configure Metadata” option.
iii) Click on “Manage Concept Dictionary”.
iv) Click on “Manage Concepts”.
v) Click on “Add New Concept”.
vi) Type in  <iframe src="https://www.csc.ncsu.edu" /> for “Fully Specified Name”, <script>alert(1);</script> for “Short Name”, and <script>alert(3);</script> for “Description”.
vii) Click on “Save Concept”.

viii) Now go to home and then "System administration" -> "Advanced administration"
ix) Click on view concepts dictionary
x) search for <script>

Before fix
ISSUE_107

After fix 
EMPT107_After

### CODE REVIEW

File: omod/src/main/webapp/dictionary/index.jsp [hunk 1]
```diff
@@ -42,7 +42,7 @@
 	//custom render, appends an arrow and preferredName it exists
 	function nameColumnRenderer(oObj){
 		if(oObj.aData[1] && $j.trim(oObj.aData[1]) != '')
-			return "<span>"+oObj.aData[0]+"</span><span class='otherHit'> &rArr; "+oObj.aData[1]+"</span>";
+			return "<span>"+oObj.aData[0]+"</span><span class='otherHit'> &rArr; "+$j("<div>").text(oObj.aData[1]).html()+"</span>";
```
Comment (HerbertYiga) [inline omod/src/main/webapp/dictionary/index.jsp:45]: thanks @Parth59   cc @isears

**Review by isears — COMMENTED** [review]
Ok, this looks good. Did you ever figure out why the patch needs to be applied to both files? It seems like a lot of code is duplicated between the two but I'm unclear on the function of each.

**Comment by Parth59** [comment]
Hi Team,

The patch required a fix to only one file. However, since the code was exactly the same in the other file, I thought making the changes in the other file will also prevent any future issues.

**Comment by sherrif10** [comment]
i agree, can you make your maven build re run again

**Review by isears — APPROVED** [review]
Yeah that sounds good to me. Thanks @Parth59 !

**Pull request merged**